xbmc/platform/android/jni/AudioFormat.h

```diff
@@ -19,6 +19,8 @@
  *
  */
 
+#define HAVE_INTTYPES_H 1
```

Comment:
see top 2 commits: https://github.com/wsnipex/xbmc/commits/aarch64

warning: this is _not_ build tested